frontend/src/assets/css/scaffolding.scss

```diff
@@ -9,7 +9,7 @@ body {
   min-height: 100vh;
   margin: 0;
   font-size: 14px;
-  font-family: Arial, sans-serif;
+  font-family: Roboto, sans-serif;
```

Comment:
it should be added to the index.html

Reply:
I don't quite get it. Am I supposed to add style tag to intex.html and here specify font-family?